core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.ts

```diff
@@ -500,7 +500,8 @@ export class TranslationModalComponent {
     if (!this.isSetOfStringDataFormat()) {
       const domParser = new DOMParser();
       const originalElements = domParser.parseFromString(
-        this.textToTranslate as string, 'text/html');
+        this.contentPanel.elementRef.nativeElement.outerHTML as string,
```

Comment:
Can you add a comment here explaining why this is necessary? Why is there a difference between `textToTranslate` and `this.contentPanel.elementRef.nativeElement.outerHTML`?

Reply:
Done